client/my-sites/comments/comment-navigation/index.jsx

```diff
@@ -324,6 +328,14 @@ export class CommentNavigation extends Component {
 	}
 }
 
+const filterBulkActions = function( value, key ) {
+	return key.indexOf( COMMENTS_CHANGE_STATUS ) !== -1 || key.indexOf( COMMENTS_DELETE ) !== -1;
+};
+
+const hasPendingActions = function( accumulator, requestState ) {
+	return accumulator || requestState.status === 'pending';
```

Comment:
`accumulator` isn't too specific. there may be more informative names for what's happening…

and actually, if we are only looking to know if such an item exists, we can use `some`

```js
filter( dataRequests, filterBulkActions ).some( ( { status } ) => status === 'pending' )
```